src/main/java/com/cleanroommc/groovyscript/compat/mods/essentialcraft/RadiatingChamber.java

```diff
@@ -0,0 +1,121 @@
+package com.cleanroommc.groovyscript.compat.mods.essentialcraft;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.helper.SimpleObjectStream;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import essentialcraft.api.RadiatingChamberRecipe;
+import essentialcraft.api.RadiatingChamberRecipes;
+import net.minecraft.item.crafting.Ingredient;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
+public class RadiatingChamber extends VirtualizedRegistry<RadiatingChamberRecipe> {
+
+    @RecipeBuilderDescription(example = @Example(".input(item('minecraft:nether_star'), item('minecraft:stone')).output(item('minecraft:beacon')).time(100).mruPerTick(10.0f).upperBalance(1.5f).lowerBalance(0.25f)"))
+    public RadiatingChamber.RecipeBuilder recipeBuilder() {
+        return new RadiatingChamber.RecipeBuilder();
+    }
+
+    @Override
+    public void onReload() {
+        removeScripted().forEach(RadiatingChamberRecipes::removeRecipe);
+        restoreFromBackup().forEach(RadiatingChamberRecipes::addRecipe);
+    }
+
+    @MethodDescription(example = @Example("item('essentialcraft:genitem', 42)"))
+    public boolean removeByOutput(IIngredient x) {
+        return RadiatingChamberRecipes.RECIPES.removeIf(r -> {
+            if (x.test(r.getRecipeOutput())) {
+                addBackup(r);
+                return true;
+            }
+            return false;
+        });
+    }
+
+    @MethodDescription(example = @Example(priority = 2000, commented = true))

```

Comment:
priority should go in `MethodDescription` annotation instead of the `Example` annotation